packages/core/src/__stampers__/api/mobile/stamper.ts

```diff
@@ -0,0 +1,78 @@
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import { generateP256KeyPair } from "@turnkey/crypto";
+import {} from "../base";
+import type { TStamp, ApiKeyStamperBase } from "@types";
+
+let Keychain: typeof import("react-native-keychain");
+
+try {
+  Keychain = require("react-native-keychain");
+} catch {
+  throw new Error(
+    "Please install react-native-keychain in your app to use ReactNativeKeychainStamper",
+  );
+}
+
+export class ReactNativeKeychainStamper implements ApiKeyStamperBase {
+  async listKeyPairs(): Promise<string[]> {
+    return await Keychain.getAllGenericPasswordServices();
+  }
+
+  async clearKeyPairs(): Promise<void> {
+    const keys = await this.listKeyPairs();
+    for (const key of keys) {
```

Comment:
tiny style nit: in a similar scenario, `clearAllSessions`, `Promise.all()` is used. any preference on approach?

Reply:
same lol. let's do `Promise.all()` then?

Reply:
there's `all()` and `allSettled()`. the first will error out if _any_ of the promises fail, which is our desired behavior I believe